usaspending_api/recipient/models.py

```diff
@@ -164,6 +164,7 @@ class SummaryAwardRecipient(models.Model):
     action_date = models.DateField(blank=True, db_index=True)
     recipient_hash = models.UUIDField(null=True, db_index=True)
     parent_recipient_unique_id = models.TextField(null=True, db_index=True)
```

Comment:
Let's remove the index on parent duns because the field is no longer being used. This field can probably be removed all together